index.rst

```diff
@@ -437,284 +329,111 @@ this case, we may have something like:
+The interface would be much clear if we rename some generic topics and attributes to reflect more closely their true meaning.
 
-Configuration mapping
----------------------
+The following is a renaming suggestion for discussion:
 
-The mapping between label, configuration file, and branch description is not
-sufficiently straightforward, especially when trying to analyse configurations
-used in the past.
-
-For instance, to perform a full reconstruction using the current framework on
-a salobj CSC, the user needs to know the configuration repository (which is
-specified in ``settingVersions.settingsUrl``), the label used to configure the
-CSC (specified in ``start.settingsToApply`` command) and the version of the
-configuration (specified in both ``settingVersions.recommendedSettingsVersion``
-and ``settingsApplied.settingsVersion``). To get the label to configuration
-file mapping the user then need to go to the configuration repository, find the
-version that matches the one used by the CSC at that time, check the
-``_label.yaml`` file so they can finally find out what configuration was used.
-
-This information should probably be present in both ``settingVersions`` and
-``settingsApplied``. For instance ``settingVersions`` should contain an
-additional entry, e.g. ``mapping``, with the names of the files the labels
-maps to (in case of git-based configuration) or any other appropriate mapping
-information (e.g. for database configuration). Then, ``settingsApplied`` should
-also have an entry for ``label`` and ``mapping`` that will contain information
-about the selected label.
-
-.. _section-appendix-available-vs-selected:
-
-Available vs. selected
-----------------------
+#.  Rename ``settingsVersions`` to ``configurationsAvailable``
 
-The ``settingsApplied`` and ``settingsVersion`` events represent what is
-available and what was selected, respectively. These topics should have
-*nearly* identical attributes; available labels vs selected label, available
-mapping vs selected map, available branch description vs. selected branch
-description and so on.
+    #.  Rename ``recommendedSettingsLabels`` to ``labels``.
+    #.  Rename ``recommendedSettingsVersion`` to ``versions``.
+    #.  Rename ``settingsUrl`` to ``url``.
+    #.  Add ``mapping``.
+        For a configuration repository ``mapping`` will reflect the content of the ``_labels.yaml`` file.
+        For a configuration database ``mapping`` will explicitly show the mapping of ``label`` to ``label:version``.
 
-.. _section-appendix-renaming:
+#. Rename ``settingsApplied`` to ``configurationApplied``
 
-Topic renaming
---------------
+    #.  Add ``label``.
+    #.  Add ``version``.
+    #.  Add ``mapping``.
+    #.  Add ``url``.
+    #.  Rename ``otherSettingsEvent`` -> ``otherInfo``.
 
-The interface would be much clear if we rename some topics and attributes to
-reflect more closely their true meaning.
+  The event will publish the selected values once the CSC is configured.
```

Comment:
Do we need to add anything about the base config that was used? if two files get used to derive the config should we not have info about both?

Reply:
so, the base config will now leave with all the other configs. So once you have the information from the repo to track down the config you automatically get everything you need.